### DISCUSSION TITLE
Cura 5.4.0.

### DISCUSSION THREAD

**Discussion by mandalorianSLEET** [community/community#67436] [Actions]
### Select Topic Area

Question

Hi why does it take so long for cura to slice a helmet it's been over half-hour it seems to do it more at night I have good internet, all Other programs are turned off , no firewall it gets to a quarter done slicing and stops

**Comment by Akash1134** [comment]
Hi @mandalorianSLEET ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
